graylog2-server/src/main/java/org/graylog/schema/SourceFields.java

```diff
@@ -20,13 +20,19 @@
     private static final String SOURCE_PREFIX = "source_";
 
     public static final String SOURCE_BYTES_SENT = "source_bytes_sent";
+    public static final String SOURCE_DEVICE_MODEL = "source_device_model";
+    public static final String SOURCE_DEVICE_VENDOR = "source_device_vendor";
     public static final String SOURCE_HOSTNAME = "source_hostname";
+    public static final String SOURCE_ID = "source_id";
     public static final String SOURCE_IP = "source_ip";
     public static final String SOURCE_IPV6 = "source_ipv6";
     public static final String SOURCE_NAT_IP = "source_nat_ip";
     public static final String SOURCE_NAT_PORT = "source_nat_port";
+    public static final String SOURCE_OS_NAME = "source_os_name";
+    public static final String SOURCE_OS_VERSION = "source_os_version";
     public static final String SOURCE_PACKETS_SENT = "source_packets_sent";
```

Comment:
@miwent Can you verify if it is:  source_packets_sent, or source_packets?  This would be the same for Destination, and the XLS has both, so want to be sure.

Reply:
Correct - it is source_packets_sent, destination_packets_sent.